sgxs-tools/Cargo.toml

```diff
@@ -33,7 +33,7 @@ path = "src/sgx_detect/main.rs"
 "sgxs-loaders" = { version = "0.2.0", path = "../sgxs-loaders" }
 "aesm-client" = { version = "0.2.0", path = "../aesm-client", features = ["sgxs"] }
 "sgx-isa" = { version = "0.3.0", path = "../sgx-isa" }
-"dcap-ql" = { version = "0.2.0", path = "../dcap-ql" }
+# "dcap-ql" = { version = "0.2.0", path = "../dcap-ql" }
```

Comment:
Please make this a [Linux-specific dependency](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#platform-specific-dependencies)